core/src/main/java/hudson/tasks/Fingerprinter.java

```diff
@@ -461,7 +461,7 @@ public AbstractBuild getBuild() {
                 }
             }
 
-            m = ImmutableMap.copyOf(m);
+            m = Collections.unmodifiableMap(m);
```

Comment:
The local variable `m` was declared within this method and goes out of scope at the end of the method, so nothing else can modify it. We do not need to copy it.